config/routes.rb

```diff
@@ -39,7 +39,9 @@
       post :saml_slo_callback, to: 'sessions#saml_slo_callback'
     end
 
-    resource :user, only: [:show]
+    resource :user, only: [:show] do
+      get :read_sso_cookie, on: :collection
+    end
```

Comment:
why is this route necessary?

Reply:
Not necessary, would delete if we keep this branch.. I just wanted to verify in the POC that we could read it back